app/controllers/checkouts_controller.rb

```diff
@@ -181,5 +179,11 @@ def checkin_bak
       end
     end
   end
+
+  # def reply
```

Comment:
We are talking to Israel to make sure to resolve our concern about security. We will remove this if security is not an issue, then we can uncomment and implement this section. If we do not finish the keyword process feature, this will be a good starting point for next year's team.